R/compiled.R

```diff
@@ -1,5 +1,5 @@
 # this does not handle LCOV_EXCL_START ect.
-parse_gcov <- function(file, package_path = "") {
+parse_gcov <- function(file, package_path = "", br = FALSE) {
```

Comment:
don't we call in somewhere `include_branches`?

Reply:
Yes, we have `include_branches` option in the print function. There might be a simpler way to achieve this. Basically, the reason why I added the `br` option to `parse_gcov` is because I wanted to call `run_gcov` with the `gcov_br` option as below 

    res <- run_gcov(pkg$path, quiet = quiet)
    res_b <- run_gcov(pkg$path, quiet = quiet, gcov_br = TRUE)

so that `res` and `res_b` are used to form the final coverage object 

  ```
coverage <- structure(c(coverage, **res**),
      class = "coverage",
      package = pkg,
      relative = relative_path,
      branches = c(branches, **res_b**))
```